GeeksCoreLibrary/Core/Services/WiserItemsService.cs

```diff
@@ -2424,14 +2424,20 @@ public async Task<List<WiserItemModel>> GetLinkedItemDetailsAsync(IWiserItemsSer
                 where.Add("(permission.id IS NULL OR (permission.permissions & 1) > 0)");
             }
 
+            var tablePrefixItemId = "";
```

Comment:
ItemId? its not a itemId right? its the prefix for the an entity, using itemId here is confusing since the term is used for linking details/links/files to items.

Reply:
I will make it itemIdTablePrefix, since itemIdEntityType is used in this piece of code for saving the entity type.